File: Java Script - Introduction to Programming/4.1. Complex Conditions/12. Volleyball.js
function volleyball([typeYear, holidayWeekend, simpleWeekend]) {
  let weekend = (48 - simpleWeekend) * 3 / 4 
  let holiday = (holidayWeekend * 2.0) / 3
  let allGames = weekend + holiday + +simpleWeekend  
  return typeYear === 'leap' ?  ~~(allGames * 115 / 100) : ~~(allGames)
}

//console.log(volleyball(['15', '30', '0']))

File: Pyrhon - Introduction to Programming/4.1. Complex Conditions/12. Volleyball.py
from math import floor

typeYear, holiday_weekend, simple_weekend = [input() for _ in range(3)]
weekend = (48 - float(simple_weekend)) * 3 / 4
holiday = (float(holiday_weekend) * 2.0) / 3
allGames = weekend + holiday + float(simple_weekend)

print(floor(allGames * 115 / 100) if typeYear == 'leap' else floor(allGames))


'''
leap
5
2
#45
'''
